tests/unit/test_lib_per_unit_provides.py

```diff
@@ -4,7 +4,7 @@
 from textwrap import dedent
 from unittest.mock import Mock
 
-from charms.traefik_k8s.v0.ingress_per_unit import IngressPerUnitProvider
+from charms.traefik_k8s.v1.ingress_per_unit import IngressPerUnitProvider
```

Comment:
`v1` is intended to mark a library stable. I would rather stick to `v0` for now if we have sufficient backwards compat.

Reply:
sure it's just my development setup, so I can have a charm using v0 requirer talk to a charm using v1 provider and check that they work :)

Reply:
Nice :-)